governing-documents/project-annual-review.md

```diff
@@ -10,81 +10,89 @@ nav_order: 5
+- The lead and secondary TAC members review the content of the PR, analyze the project for community health indicators, and place their findings into a thread in the TAC channel (**LINK TBD:** `#tac`) for discussion.
+- The project's TSC and advocates are invited to the public TAC meeting to engage in a TAC-led discussion about the project. Project TSC members are not obligated to attend, but it is highly recommended.
+- The lead TAC member provides a summary of the project and leverages the thread's content as the basis of discussion. The discussion typically focuses on what is going well with the project and areas to improve.
+- The project's TSC and contributors are invited to use this time to voice any concerns and requests for help they may have that are not captured in the PR (or highlight asks within the PR).
+- At the conclusion of the public meeting, the TAC votes to approve the annual review, including any change in the project's status. Should a concern be registered on a project, the vote will be held at a later date.
+- After the meeting wraps up, the lead TAC member should summarize the discussion on the PR in the form of a comment to document information for the project and community.
 
 ### Filing an Annual Review
-Hyperledger Foundation staff will notify the project maintainers and copy the TAC when the project review is due. 
 
-Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports).
+LF Decentralized Trust staff will notify the project TSC and copy the TAC when the project review is due.
 
-* Raise a PR titled *[year] [Project Name] Annual Review* (e.g., 2024 Hyperledger Amazing Annual Review)
-* The PR should include a file called `./<year>/<year>-annual-<Project-Name>.md` (e.g., `2024/2024-annual-Hyperledger-Amazing.md`) using [the annual review template](../project-reports/0000-annual-review-template.md)
-    - Update the information at the top of the file:
-        - change the `title` line to `YYY Annual Review Project Name` (e.g., `title: 2024 Annual Review Hyperledger Amazing`)
-        - change the `parent` line to `YYYY` (e.g., `parent: 2024`)
-        - change the `grand_parent` to `Project Updates` (i.e., `grand_parent: Project Updates`)
-        - remove the `nav_exclude` line
-    - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
-* Send an email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+Project TSC members are responsible for agreeing between them who will complete the annual review. One of the TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
```

Comment:
Q: Should the github link be a TODO?

Reply:
The links can be checked by a link checker when the new repo is used.  I don’t think we need a todo on each where we probably have it right.  I did put a couple of “TBDs” where I didn’t know at all what the link would be.